apps/condo/jest.config.js

```diff
@@ -45,7 +45,10 @@ module.exports = {
             // NOTE: need to pass uuid export syntax through babel
             transformIgnorePatterns: ['/node_modules/(?!(uuid|nanoid|msgpackr|n2words)/)'],
             moduleNameMapper: {
-                '@open-condo/(.*)': '<rootDir>/../../packages/$1',
+                '^@open-condo/(.*?)/(.*)$': [
+                    '<rootDir>/../../packages/$1/$2',
```

Comment:
wtf? you definetly dont need this. you're doing something wrong by transpliling entire packages dir